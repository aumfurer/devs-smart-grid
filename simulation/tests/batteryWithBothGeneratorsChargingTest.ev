01:00:00:00 radiation 50
04:00:00:00 wind_speed 50