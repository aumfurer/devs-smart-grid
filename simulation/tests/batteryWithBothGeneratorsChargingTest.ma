[top]

components : bateria@bateria celda_solar@celda_solar windTurbine@generador_eolico

out : out_port

in : radiation wind_speed

link : radiation solar_change@celda_solar
link : wind_speed wind_change@windTurbine

link : out@celda_solar solar_panel@bateria 
link : out@windTurbine wind_turbine@bateria 

link : battery_state@bateria out_port
