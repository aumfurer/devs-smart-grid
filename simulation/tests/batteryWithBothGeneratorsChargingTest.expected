03:00:00:000:0 out_port      1.00000
07:30:00:000:0 out_port      2.00000
